cmd/iamctl/minify.go

```diff
@@ -1,30 +1,95 @@
 package main
 
 import (
+	"bytes"
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
 	"sort"
 	"strings"
+	"text/template"
 )
 
+// compressionPrefixes defines a list of action prefixes in the policy
+// that are going to be compressed using wildcards.
+var compressionPrefixes = map[string]string{
+	"ec2:Describe":                  "ec2:Describe*",
+	"elasticloadbalancing:Describe": "elasticloadbalancing:Describe*",
+}
+
+func generateIAMPolicy(inputFilename, output, pkg string) {
```

Comment:
Should `generateIAMPolicy` really be in `minify.go`?

Reply:
Yeah, doesn't make sense. Didn't want to have too many files. Two options, 
1. Rename file to `policy.go`
2. Drop it in a new file `policy.go`

Reply:
Fixed.